consensus/src/dag/types.rs

```diff
@@ -503,39 +503,78 @@ impl BroadcastStatus for CertificateAckState {
 #[derive(Serialize, Deserialize, Clone, Debug)]
 pub struct RemoteFetchRequest {
     target: NodeMetadata,
-    start_round: Round,
-    exists_bitmask: Vec<Vec<bool>>,
+    exists_bitmask: DagSnapshotBitmask,
```

Comment:
we can just make the target a Vec<NodeMetadata> and only put the missing parents there

Reply:
I think we should send all parents metadata to check the voting power of the parents on the receiver-side, just as a sanity check. Otherwise, any validator can draft RemoteFetchRequests with one parent and make a validator work more, send more data and consume bandwidth.

Reply:
I think that's orthogonal problem to solve (waste resource/bandwidth), I can always send a remote fetch request from any valid node to achieve the same effect